schemas/org.gnome.shell.extensions.pop-shell.gschema.xml

```diff
@@ -33,6 +33,11 @@
             <summary>Show title bars on windows with server-side decorations</summary>
         </key>
 
+        <key type="b" name="show-stack-tab-buttons">
+            <default>true</default>
+            <summary>Allow showing tab buttons for stacked windows</summary>
```

Comment:
"Allow showing" is a roundabout way to phrase this. Nothing is being "allowed." The verb should just be "show."

"Tab buttons" is also potentially confusing, since it's unclear if that refers to the tabs themselves or buttons on the tabs (e.g. the close buttons.)

Please simplify this to `Show tabs for stacked windows`.

Reply:
Ah ok :) Maybe confusion on my side. I understood the Tab Interface to hold both the app's window alongside the button at the top. Therefore I "over"specified it in this way :)